ide/o.apache.xml.resolver/build.xml

```diff
@@ -38,7 +38,7 @@
         <FileCRC32Calculator file="external/xml-resolver-1.2.jar" property="o.apache.xml.resolver.crc32" />
 
         <echo message="Patching xml-resolver-1.2-sources.jar for Netbeans..." />
-        <taskdef name="patchfile" classname="org.netbeans.nbbuild.extlibs.PatchFile" classpath="${nbantext.jar}"/>
+        <taskdef name="ide/patchfile" classname="org.netbeans.nbbuild.extlibs.PatchFile" classpath="${nbantext.jar}"/>
```

Comment:
Travis says
BUILD FAILED
/home/travis/build/apache/incubator-netbeans/nbbuild/templates/projectized.xml:99: The following error occurred while executing this line:
/home/travis/build/apache/incubator-netbeans/nbbuild/templates/projectized.xml:99: The following error occurred while executing this line:
/home/travis/build/apache/incubator-netbeans/nbbuild/templates/projectized.xml:99: The following error occurred while executing this line:
/home/travis/build/apache/incubator-netbeans/ide/o.apache.xml.resolver/build.xml:49: Problem: failed to create task or type patchfile
Cause: The name is undefined.

----------------------------------------------------------------
this may be due to the name change of this taskdef
from 
patchfile
to 
ide/patchfile